packages/block-editor/src/components/off-canvas-editor/block.js

```diff
@@ -331,10 +332,14 @@ function ListViewBlock( {
 						}
 					>
 						{ () => (
-							<BlockEditButton
-								label={ editAriaLabel }
-								clientId={ clientId }
-							/>
+							<>
+								{ isEditable && (
+									<BlockEditButton
+										label={ editAriaLabel }
+										clientId={ clientId }
+									/>
```

Comment:
```suggestion
									<BlockEditButton
									      	{ ...treeGridCellProps }
										label={ editAriaLabel }
										clientId={ clientId }
									/>
```

I think we may need to spread the props from TreeGridCell here to make the button accessible by keyboard?

Reply:
https://developer.wordpress.org/block-editor/reference-guides/components/tree-grid/#children-as-a-function

> Props passed as an argument to the render prop must be passed to the child focusable component/element within the cell. If a component is used, it must correctly handle the onFocus, tabIndex, and ref props, passing these to the element it renders. These props are used to handle the roving tab index functionality of the tree grid.

Reply:
@ajlende guided me through this and we refactored the `BlockEditButton` to accept the props from `TreeGridCellProps`